libs/particles/particles.ts

```diff
@@ -341,6 +341,7 @@ namespace particles {
     //% blockId=particlesremoveeffect block="remove effects on %anchor=variables_get(mySprite)"
     //% group="Effects"
     export function removeEffects(anchor: ParticleAnchor) {
+        if (!this.sources) return;
```

Comment:
this does not exist here

Reply:
Yup, noticed this typo after I committed :)